### DISCUSSION TITLE
pay out

### DISCUSSION THREAD

**Discussion by GrendalKahn** [Chia-Network/chia-blockchain#15748] [Support]
on the transaction window I stopped getting paid out July 1st?

**Comment by BrandtH22** [comment]
Hey Grendal, have you made sure that your chia client is up to date and that the node is still in sync?

Also a bit more context will be helpful:
Are you solo or pool farming?
Have you verified whether plots are passing the plot filter / submitting partials to the pool?

**Comment by GrendalKahn** [comment]
I updated my Chia client and that seems to have resolved the issue. I am
guessing it created the space.pool account issue I had. As space.pool
claimed I was getting a 100% partials. Meaning it was not giving me the
daily rewards I was getting before. I came back from my July vacation and
it was not giving rewards. The solo mining I was doing was working just
fine. I just wish there was a way for it to alert you that partials were an
issue. Thanks for responding to the thread I created so fast. I wish I
could have responded to you sooner. But I was sidetracked with my GPU
rigges having an issue after I went on vacation as well.

**Comment by BrandtH22** [comment]
Hey @GrendalKahn , thank you very much for the detailed response and overview of the resolution that you found.
You might want to relay that potential enhancement to the spacescan team so they can see if it is possible for them to enable notifications for farms that either go offline or submit below a certain percent of valid partials, that is an excellent idea and I know others would also benefit.

As for the chia side of things there is an official dashboard (chiaExporter - https://github.com/Chia-Network/chia-exporter) and I know some members of the community have setup alerts based on those repos but I am not aware of an official system for remotely being notified of partials submissions.

Since the main issue has been resolved I will be closing out this ticket but do not hesitate to reach out with any further questions or comments :+1:
